pydm/main_window.py

```diff
@@ -370,6 +372,10 @@ def open(self, filename, macros=None, args=None, target=None):
             self.ui.actionEdit_in_Designer.setText(edit_in_text)
             if (self.designer_path and ui_file) or (py_file and not ui_file):
                 self.ui.actionEdit_in_Designer.setEnabled(True)
+
+        # Create and run the thread for establishing the channel connections which have been queued above
+        establish_connections_thread = threading.Thread(target=data_plugins.establish_queued_connections, daemon=True)
+        establish_connections_thread.start()
```

Comment:
A second thought, unrelated to the first:
We don't use the main window in our typhos stuff but may still want to benefit from the change. 
* Is there another spot that PyDM could place this establish connection logic?
* Could there be a function that displays (?) call on initialization (or elsewhere) effectively saying "start a connection queue thread in the background if there isn't one already running"
* Regardless of whether the function described above is used on init, I think it would be a general purpose function that should be implemented and used in the main window